src/tools/info/ucx_info.c

```diff
@@ -28,6 +28,7 @@ static void usage() {
     printf("  -y              Show type and structures information\n");
     printf("  -s              Show system information\n");
     printf("  -c              Show UCX configuration\n");
+    printf("  -i              Print UCX configuration in sample INI file format\n");
```

Comment:
i think the only difference is that default configuration is commented out?
how about
`-C    Comment-out default configuration values`
and
UCS_CONFIG_PRINT_COMMENT_DEFAULT